pkg/k8s/apps/crud_test.go

```diff
@@ -266,3 +267,225 @@ func TestValidateMountPaths(t *testing.T) {
 	}
 
 }
+
+func TestListDevModeOn(t *testing.T) {
```

Comment:
Can we test that we are not getting one dev container (a deployment in dev mode) that is not defined in the manifest?

Reply:
added `"dev-is-not-at-manifest"`